### PR TITLE
Beef up docs landing page

### PR DESCRIPTION
**Pull request by dannygoldstein**
This PR beefs up the landing page for the docs, adding:

1. A SkyPortal logo to the upper left corner
2. A responsive mockup showing SkyPortal in use on a desktop and a mobile device, showcasing some of its features
3. A Quick Start section showing users who want to "just run" how to launch their own demo instance
4. Status badges (Github, tests, license, joss).

I also broke out some of the text from `about.md` to `index.rst`. I did not alter any of the text body.

### CODE REVIEW

**Comment by dannygoldstein** [comment]
Screen Shot 2020-08-17 at 4 32 45 PM

**Comment by stefanv** [comment]
@dannygoldstein Thanks, this looks great.  Do you think it is accurate to say that we have filtering capability?  Otherwise, I think this can go in.

**Comment by stefanv** [comment]
The current repo requires 3.2 MB to download.  The new image added is 0.5MB.  Could we downscale and use JPEG to make it a bit smaller?

**Comment by dannygoldstein** [comment]
@stefanv I removed the stuff on filtering, and downsized the responsive image from 505kb to 209kb JPG.

**Comment by profjsb** [comment]
Suggested changes: 
 1) Make the responsive image a gif, with a transparent background around the computer/mobile images; remove the margins (the empty space currently takes up a lot of page space).
 2) The motivating text is not the way I would explain SkyPortal. I will generate some text and send it offline.

**Comment by dannygoldstein** [comment]
[ch1195]

**Comment by profjsb** [comment]
Here's a reasonable blurb for the landing page @dannygoldstein:

**TL;DR**: SkyPortal is a standalone extensible, API-first web application that can be used to serve public and/or private data on infrastructure that you manage. Tired of managing observations of thousands of sources for you and your group in Google Docs? Got a new astronomical survey you'd like to serve to the public and don't want build and maintain a website yourself? Want a place to manage requests for observations during an upcoming observing run? Sometimes work in a mobile/tablet environment? Then this project may be for you!

SkyPortal is a fully open-source portal to manage and collaborate on time-domain sources and events. On the backend, it has been designed to ingest and store millions of events per night from disparate discovery streams. The frontend allows searching by source properties, source-level summary pages, synchronous and asynchronous source-level communication, and group and user management. It also contemplates headless bi-directional interactions with 3rd-party programmatic enrichment tools: webhooks and APIs allow, for instance, machine-learning classifiers to opine on new events as they arrive.

By design SkyPortal makes use of hierarchical access control, frontend/backend abstractions with strong API contracts, elastic scaling, modern CI/CD practices, logging for auditability and traceability, and documentation meant to foster extensibility. It is built to run at the individual laptop level, in on-premise clusters, or in the cloud. The initial use cases were developed in close consultation with other time-domain practitioners and studying and using proprietary portals.

**Comment by dannygoldstein** [comment]
@profjsb @stefanv all changes implemented 

  
Screen Shot 2020-08-18 at 11 53 36 PM

**Comment by stefanv** [comment]
Thanks, Danny.  I'm a bit hesitant to put the "test suite passing" badge on the documentation (because, you know, test suites fail from time to time).  Otherwise I am +1 on all this... thanks very much!

**Comment by profjsb** [comment]
@dannygoldstein -- looks great. I agree with Stefan to remove the tests badge (for now). One last request: please add in (at the bottom of the landing page) and acknowledgement to the Moore foundation (see the homepage of skyportal.io): "We thank the Gordon and Betty Moore Foundation for a Software SkyPortal grant which partially covered the cost of the development of this project."

**Comment by dannygoldstein** [comment]
@profjsb @stefanv done
